webroot/src/components/PaymentProcessorConfig/PaymentProcessorConfig.vue

```diff
@@ -0,0 +1,34 @@
+<!--
+    PaymentProcessorConfig.vue
+    CompactConnect
+
+    Created by InspiringApps on 12/5/2024.
+-->
+
+<template>
+    <Card class="payment-processor">
+        <div class="payment-processor-form">
+            <h2>{{ $t('compact.paymentProcessorCredentials') }}</h2>
+            <MockPopulate :isEnabled="isMockPopulateEnabled" @selected="mockPopulate" />
+            <div class="payment-processor-form-container">
+                <form @submit.prevent="handleSubmit">
+                    <InputText :formInput="formData.accountId" />
+                    <InputPassword
+                        :formInput="formData.accountKey"
+                        :joiMessages="joiMessages"
+                        :showEyeIcon="true"
+                        :showRequirements="false"
+                    />
+                    <InputSubmit
+                        :formInput="formData.submit"
+                        :label="submitLabel"
+                        :isEnabled="!isFormLoading"
```

Comment:
Do we want :isEnabled="!isFormLoading || isFormValid"?

Reply:
On other forms, the designs have called for the submit button to always be enabled and trigger the validation errors if invalid. This is just following that for consistency.